Makefile

```diff
@@ -9,13 +10,20 @@ JOB_NAME = 1
 
 all: supervisor
 
-IMAGE = "resin/$(ARCH)-supervisor:$(SUPERVISOR_VERSION)"
-SUPERVISOR_IMAGE=$(DEPLOY_REGISTRY)$(IMAGE)
-
 PUBNUB_SUBSCRIBE_KEY = sub-c-bananas
 PUBNUB_PUBLISH_KEY = pub-c-bananas
 MIXPANEL_TOKEN = bananasbananas
 
+ifeq ($(BASE_DISTRO),alpine)
+	IMAGE = "resin/$(ARCH)-supervisor:$(SUPERVISOR_VERSION)-$(BASE_DISTRO)"
+	DOCKERFILE = alpine.$(ARCH)
+else
```

Comment:
I think it would be better to change this to a not exists check, as it stands if I specify `BASE_DISTRO=fedora` for instance then it will just silently use a debian base without any warning/error